src/System.Collections.NonGeneric/tests/System.Collections.NonGeneric.Tests.csproj

```diff
@@ -147,14 +147,17 @@
     <Compile Include="$(CommonTestPath)\System\Diagnostics\DebuggerAttributes.cs">
       <Link>Common\System\Diagnostics\DebuggerAttributes.cs</Link>
     </Compile>
-  </ItemGroup>
-  <!-- Performance Tests -->
-  <ItemGroup Condition="'$(RunPerfTestsForProject)' == 'true'">
+    <!-- Performance Tests -->
     <Compile Include="Performance\Perf.HashTable.cs" />
     <Compile Include="$(CommonTestPath)\System\PerfUtils.cs">
       <Link>Common\System\PerfUtils.cs</Link>
     </Compile>
   </ItemGroup>
+  <!-- Optimizations to configure Xunit for performance -->
+  <ItemGroup Condition="'$(Performance)' == 'true'">
```

Comment:
Ah, I see. I don't think it will matter if the assemblies have extra assembly attributes on them (they don't really do anything unless you are looking for them), but not being able to compile them might be an issue :smile: . I think an okay workaround is to just add a <ContainsPerfTests>true</> property and keep this assembly info stuff in the common properties. Does that make sense?